test/lifecycle.test.js

```diff
@@ -38,15 +38,20 @@ test("oninsert", done => {
 })
 
 test("onupdate", done => {
+  const mock = jest.fn()
   app({
     state: 1,
     view: state =>
-      h("div", {
-        onupdate: e => {
-          expect(state).toBe(2)
-          done()
-        }
-      }),
+      h("div", { onupdate: mock },
+        h("div", {
+          class: state,
+          onupdate: element => {
+            expect(mock).not.toHaveBeenCalled()
+            expect(state).toBe(2)
+            done()
+          }
+        })
```

Comment:
@lukejacksonn Are you sure about this test? If you call the action inside loaded, it will be called after the next cycle because of how we are faking rAF with setTimeout.